doc/source/getting-started/installation.rst

```diff
@@ -197,26 +168,45 @@ please do:
 _______________________
 
 
-.. _installing-pytorch:
+.. _installing-qml-backends:
 
-pytorch
-^^^^^^^
+Backends with automatic differentiation support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need automatic differentiation support, for tracing gradients of your
+quantum algorithm or for building some quantum machine learning routine,
+the right backends for you are those provided by the `Qiboml <https://github.com/qiboteam/qiboml>`__
+package.
 
-If the `PyTorch <https://pytorch.org/>`_ package is installed Qibo
-will detect and provide to the user the possibility to use ``pytorch``
-backend.
+In particular, Qiboml currently support `Pytorch <https://pytorch.org/>`_ and
+`Tensorflow <https://www.tensorflow.org>`_ interfaces, integrating the qibo functionalities
+into these well-known machine learning frameworks. Quantum layers can be constructed
+and added to your Pytorch or Tensorflow models, and trained using any supported
+optimization routine.
 
-In order to switch to the ``pytorch`` backend please do:
+In order to use these quantum machine learning backends please make sure the
+preferred package is installed following `Tensorflow's <https://www.tensorflow.org/install>`_
+or `Pytorch's <https://pytorch.org/get-started/locally/>`_ installation instructions.
+
+To switch to Tensorflow or Pytorch backend please do:
```

Comment:
Not sure whether it is worth to have details about the ml backends. I would probably place this in the `qiboml` docs and link them here.

Reply:
I agree, I would remove these lines. Probably it is better to wait the moment the Qiboml docs are ready so that we can just link to the proper section.